src/components/AdminMaintenance/EmSubmissionPopOut/EmSubmissionPopout.js

```diff
@@ -13,7 +13,7 @@ const getDateString = (date) => {
   return dArr[0]
 }
 
-export const EmSubmissionModal = ({ showModal, close, isOpenModal, isExtendModal, isCloseModal, isApproveModal, openDate, closeDate, selectedRows, setReloadTableData }) => {
+export const EmSubmissionModal = ({ showModal, close, isOpenModal, isExtendModal, isCloseModal, isApproveModal, openDate, closeDate, selectedRows, setReloadTableData, reportingPeriods }) => {
```

Comment:
```suggestion
export const EmSubmissionModal = ({ showModal, close, isOpenModal, isExtendModal, isCloseModal, isApproveModal, selectedRows, setReloadTableData, reportingPeriods }) => {
```

We don't need openDate and closeDate prop since we are passing selected rows. 

Please update openDate and closeDate based on the selectedRow on extend modal.